### ISSUE TITLE
OrientDB database import failing with java.lang.OutOfMemoryError

### ISSUE THREAD

**Issue by chaitanyachuri** [orientechnologies/orientdb#6116]
Hi,

I've been trying to import a database created on a linux machine using version 2.1.11. This original database export was created using the "EXPORT DATABASE" command on an identical machine using the same version and the same memory allocations for the JVM. 

I am now trying to import this to a new machine with the same version of OrientDB with more memory allocations to the process than on the original machine that created the export. I follow the following steps to do the import
1. Log into console.sh
2. CREATE DATABASE <databasename> <user> <password>
3. IMPORT DATABASE <databaseexportfile> -preserveClusterIDs=true

However the process always ends with the following error 

Error: java.lang.OutOfMemoryError: Java heap space

There seems to be some sort of memory leak in this process and increasing the memory allocation in both server.sh and console.sh do not help. It always fails at the same level of the import.

Thanks in advance!

**Comment by laa** [comment]
@chaitanyachuri This issue should be fixed in 2.2 rc1, could you try ?

**Comment by lvca** [comment]
@laa do you know the cause of this?

**Comment by laa** [comment]
@lvca I will not dare to tell till I will not get feedback for @chaitanyachuri . But we have already fixed similar issue.

**Comment by laa** [comment]
@lvca this issue is already assigned to "storage team" some of us will take it from queue during weekly bug fixing phase.

**Comment by chaitanyachuri** [comment]
Hi @laa and @lvca . Thank you for your replies.
We tried using the 2.2 rc1 but after we installed that, we couldn't even open up OrientDB Studio to log into our databases. The GUI wouldn't open up. When we downgraded back to 2.1.11, the Studio UI came back. So I couldn't even test my imports. Is this a known issue? 
I'm a little apprehensive about relying on an RC version to develop my work if I can't look at my database schema. Our work relies heavily on the Orient Studio interface.
